src/adfh/ADFH.c

```diff
@@ -2913,6 +2964,8 @@ void ADFH_Put_Dimension_Information(const double   id,
   int i, swap = 0;
   hsize_t new_dims[ADF_MAX_DIMENSIONS];
   char new_type[3];
+  hid_t g_propdataset_bk;
+  int filter_set;
```

Comment:
Initialize to zero?   It is done down at line 3059, but hard to see what is happening between here and there.  Should be no disadvantage to initializing here.

Reply:
good point, fixed.